src/main/java/robot/subsystems/drivetrain/pure_pursuit/Path.java

```diff
@@ -275,14 +293,18 @@ public void generateSmoothing(double weight_data, double weight_smooth, double t
     }
 
     /**
-     * Attributes to all points their distance from the start.
+     * Attribute to all points their distance from the start.
+     * sets the distance of each point from the start and saves as a parameter, so that it doesnt have to be calculated real time.
```

Comment:
Use the imperative form. Also, a few tweaks.
```suggestion
     * Set the distance of each point from the start and save as a parameter, so that it doesn't have to be calculated real time.
```